tests/contracts/token_fixtures.py

```diff
@@ -27,7 +27,7 @@ def ganachecli_command():
     if "GANACHE_BINARY" in os.environ:
         return os.environ["GANACHE_BINARY"], 5
     else:
-        return 'ganache-cli', 2
+        return 'ganache-cli', 8
```

Comment:
I assume thats a mistake, you are not testing this in ganache :-)

Reply:
You are right, this should be reverted :)